### ISSUE TITLE
A11y_AzureDataStudioExtensionAzureSQLMigration_Migratetoazuresql_MigrationMode_ScreenReader: ORCA screen reader is unnecessary announcing text unselected when we navigate to "Resource group" and "Azure database migration service" dropdown.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23898]
### Actual Result:
ORCA screen reader is unnecessary announcing 'text unselected' when we navigate to "Resource group" and "Azure database migration service" dropdown.

### Expected Result:
ORCA screen reader should not announce unnecessary 'text unselected' when we navigate to "Resource group" and "Azure database migration service" dropdown.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #Benchmark; #ORCA; #WCAG1.3.1; #FTP; #SH-AzureDataStudioExtensionAzureSQLMigration-Linux-Jul23; #SQL Migration extension for Azure Data Studio; #Linux; #Element:Dropdown;

### Environment Details:
Application Name: Azure Data Studio extension 'Azure SQL Migration'
Operating System: Ubuntu 20.04.2 LTS
Azure data studio 1.44.1
Azure SQL Migration v1.4.8
Screen Reader: ORCA

### Repro Steps:

1. Install Azure data studio application and open it
2. TAB till Extension press ENTER key and install Azure SQL Migration extension.
3. TAB till Respective server and right click. TAB till manage and press ENTER key.
4. TAB till Azure SQL Migration and press ENTER key. TAB till "migrate to azure sql" and press ENTER.
5. Select start new session and fill all the data and press ENTER on next button.
6. Select the respective databases and ENTER on next button.
7. Fill all the necessary details and press ENTER key on Next button till "Azure database migration service" page appears.
8. Start ORCA screen reader. TAB to "Resource group" and "Azure database migration service" dropdown and observe the issue.

### User Impact:
The screen reader users will get incorrect information and will get confused whether the control is selected or not.

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Attachment:

[23898_Issue.webm](https://github.com/microsoft/azuredatastudio/assets/90901519/a86bedcf-78e0-4bb1-9ac8-d1a23adff0d3)

**Comment by krritik** [comment]
@PoojaNamde , I checked the issue in Ubuntu 20.04. I do not see it now. Can you verify if the issue is still there.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:09-11-23;
Verified this issue in the Azure data studio 1.46.0 version. Issue is fixed hence closing the bug.
